website/docs/plugin/best-practices/sensitive-state.mdx

```diff
@@ -0,0 +1,53 @@
+---
+page_title: Plugin Development - Sensitive State Best Practices
+description: Recommendations for handling sensitive information in state.
+---
+
+# Handling Sensitive Values in State
+
+Many organizations use Terraform to manage their entire infrastructure, and
+it's inevitable that [sensitive information will find its way into
+Terraform](/terraform/language/state/sensitive-data) in these circumstances. There are a
+couple of recommended approaches for managing sensitive state in Terraform.
+
+## Using the `Sensitive` Flag
+
+When working with a field that contains information likely to be considered
+sensitive, it is best to set the
+[`Sensitive`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/resource/schema#Attribute)
```

Comment:
If we're going to link to the framework Go documentation here, I think its best we frame this sentence around "enabling the sensitive flag in the SDK the provider is written in", where the framework is one option. We can also link to the sdk Go documentation.